### PR TITLE
[SDPTA-868] Remove unwanted slack notifications

### PR DESCRIPTION
**Pull request by tim-yao**
As a dev/test engineer, I want no slack notifications to the sdp-test-report channel as the regression testing for a feature PR can be checked on the PR itself by developers, so it won’t send additional messages to the sdp-test-report channel.
Also fixed the slack notification test type emojis.

### CODE REVIEW

**Review by FleetAdmiralButter — APPROVED** [review]

File: .github/workflows/run_e2e_be.yml [hunk 1]
```diff
@@ -150,7 +150,19 @@ jobs:
...
+        ( inputs.slack_notify == 'release_only' &&
+          ( startsWith(github.ref_name, 'release/') ||
+            github.ref_name == 'uat_test' ||
+            github.ref_name == 'uat' ||
+            github.ref_name == 'master' ||
+            github.ref_name == 'production' ||
+            github.ref_name == 'standby'
+          )
+        )
+      )
```
Comment (tim-yao) [inline .github/workflows/run_e2e_be.yml:165]: @FleetAdmiralButter @MarkCalvert @steveworley do you know any better way to write it?

Comment (steveworley) [reply .github/workflows/run_e2e_be.yml:170]: I think you can use 

```
contains(fromJSON('["push", "pull_request"]'), github.event_name)
```

https://docs.github.com/en/actions/learn-github-actions/expressions#example-matching-an-array-of-strings

Comment (tim-yao) [reply .github/workflows/run_e2e_be.yml:170]: @steveworley that's exactly what I am looking for! Thanks!

Comment (tim-yao) [reply .github/workflows/run_e2e_be.yml:170]: cc @klaukl

**Comment by tim-yao** [comment]
@FleetAdmiralButter can you please review this? It has been tested and can be merged.

**Review by steveworley — APPROVED** [review]

**Pull request merged**